src/lib/components/HeadingIcon/HeadingIcon.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './HeadingIcon.css';
+
+const HeadingIcon = props => (
+  <div className="p-heading-icon">
+    <div className="p-heading-icon__header">
+      <img className="p-heading-icon__img" alt="" src={props.imageSrc} />
```

Comment:
`props.title` should be passed as an alt value or an additional prop added for the alt value.